tests/e2e/isbservice.go

```diff
@@ -413,30 +418,35 @@ func updateISBServiceRollout(
 		Expect(err).ShouldNot(HaveOccurred())
 	}
 
-	verifyISBServiceSpec(Namespace, isbServiceRolloutName, verifySpecFunc)
+	VerifyISBServiceSpec(Namespace, isbServiceRolloutName, verifySpecFunc)
 
 	verifyISBSvcRolloutReady(isbServiceRolloutName)
 	verifyISBSvcReady(Namespace, isbServiceRolloutName, 3)
 
-	verifyInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyNoOp)
-	verifyPipelineRunning(Namespace, pipelineRolloutName, true)
+	VerifyInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyNoOp)
+	// VerifyPipelineRunning(Namespace, pipelineRolloutName)
```

Comment:
remove old line

Reply:
oh wait, actually, I guess the logic below still needs to be updated not to use the pipeline name to determine if it's failed, right? I guess this was in the code like that a few places?

Reply:
Ack, missed that. Will change.